vendor/github.com/cilium/ebpf/map.go

```diff
@@ -1090,7 +1090,7 @@ func (m *Map) Clone() (*Map, error) {
 // You can Clone a map to pin it to a different path.
 //
 // This requires bpffs to be mounted above fileName.
-// See https://docs.cilium.io/en/stable/concepts/kubernetes/configuration/#mounting-bpffs-with-systemd
+// See https://docs.cilium.io/en/stable/network/kubernetes/configuration/#mounting-bpffs-with-systemd
```

Comment:
This file (and all others in `vendor/`) are vendored from other packages. These files need to be changed in the respective upstream repository (i.e. `github.com/cilium/ebpf` and `k8s.io/api/core/v1`) first and then vendored back into this repo.

Reply:
Thanks for the info. https://github.com/cilium/ebpf/pull/942